src/lib/helpers.js

```diff
@@ -24,4 +25,20 @@ async function step(str, fn) {
   }
 }
 
+export async function isMinaGraphQlEndpointAvailable(endpoint) {
+  try {
+    const response = await fetch(endpoint, {
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json' },
+      body: JSON.stringify({ query: '{ syncStatus }' }),
+    });
+    if (!response.ok) {
+      return false;
+    }
+    return true;
+  } catch (error) {
+    return false;
```

Comment:
Do you think it is worth surfacing the error here @shimkiv ?

Reply:
Log stringified error now.

Reply:
After looking at the output, I think we should remove this log. I missed that you already threw an error if the result of the `isMinaGraphQlEndpointAvailable` is false.

Reply:
Reverted.